### PR TITLE
:sparkles: Restore find pool page

### PR DESCRIPTION
**Pull request by KipandaJr**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206531706443143

### CODE REVIEW

File: pages/pools/find.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        value: '0',
+        ...values(COINS[network])[1],
+      },
+    },
+  });
+
+  return (
+    <FormProvider {...form}>
+      <ModalProvider>
+        <SEO pageTitle="Find Pool" />
```
Comment (MarioBatalha) [inline pages/pools/find.tsx:32]: The page title has changed on Figma to Create Pool

File: views/pools/header/action-group.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ const ActionGroup: FC<ActionGroupProps> = ({ showSearchView }) => {
             </Box>
           }
         >
-          Create Pool
+          Find Pool
```
Comment (MarioBatalha) [inline views/pools/header/action-group.tsx:49]: Changed to Create Pool

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: pages/pools/find.tsx (hunk 1 above)
Comment (KipandaJr) [reply pages/pools/find.tsx:32]: We change to Find Pool... Figma not updated

File: views/pools/header/action-group.tsx (hunk 2 above)
Comment (KipandaJr) [reply views/pools/header/action-group.tsx:49]: We change to Find Pool... Figma not updated

**Review by MarioBatalha — APPROVED** [review]

File: views/find-pool/find-pool-form/find-token-form.validation.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import * as yup from 'yup';
```
Comment (git-marcopitra) [inline views/find-pool/find-pool-form/find-token-form.validation.ts:1]: We don't need this file

File: views/find-pool/find-pool-form/index.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import { Box, Button, Form, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import CreatePoolFormSelectDex from './find-pool-form-select-dex';
```
Comment (git-marcopitra) [inline views/find-pool/find-pool-form/index.tsx:4]: `Create` > `Find`

File: views/find-pool/find-pool-form/index.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+import CreatePoolFormSelectToken from './find-pool-form-select-token';
```
Comment (git-marcopitra) [inline views/find-pool/find-pool-form/index.tsx:5]: `Create` > `Find`

**Review by git-marcopitra — CHANGES_REQUESTED** [review]
And when select the icon put it on url

File: pages/pools/find.tsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+import { SEO } from '@/components';
+import { COINS } from '@/constants/coins';
+import { ModalProvider } from '@/context/modal';
+import { useNetwork } from '@/context/network';
+import FindPool from '@/views/find-pool';
+import { IPoolForm } from '@/views/find-pool/find-pool.types';
+
+const FindPoolPage: NextPage = () => {
+  const { network } = useNetwork();
+  const form = useForm<IPoolForm>({
```
[the remark this answers is not in the document]
Comment (KipandaJr) [reply pages/pools/find.tsx:14]: I'll need a little help to apply this and this _`And when select the icon put it on url`_

**Comment by git-marcopitra** [comment]
@KipandaJr there is a problem on find pool form

File: views/find-pool/find-pool-form/find-pool-form-select-dex.tsx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+          border="2px solid"
+          borderColor="outline"
+        >
+          {DEX_LIST.map((dexName) => (
+            <ListItem
+              key={v4()}
+              width="100%"
+              title={dexName}
+              cursor="pointer"
+              onClick={() => {
```
Comment (atilasousa) [inline views/find-pool/find-pool-form/find-pool-form-select-dex.tsx:62]: create a handleClick for this one

**Review by atilasousa — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**